src/components/dapp-staking/stake-manage/SelectFunds.vue

```diff
@@ -14,12 +14,12 @@
           <img :src="list.iconUrl" :alt="list.name" class="item-logo" />
           <span class="text--item-name">{{ list.name }}</span>
           <div class="column--amount">
-            <span>{{
-              $t('amountSymbol', {
-                amount: $n(truncate(ethers.utils.formatEther(list.balance))),
-                symbol: nativeTokenSymbol,
-              })
-            }}</span>
+            <span>
+              <TokenBalance
```

Comment:
![Screenshot from 2022-10-11 11-24-50](https://user-images.githubusercontent.com/40356749/194983248-de500ade-2191-446d-b6d3-0931b448bcfb.png)

Please correct me if I'm wrong, but why can I select different tokens to stake with even though the chain can only support native tokens? Plus, these tokens are ERC20 while I'm logged in with my native account,

The token list also says `Select Chain`

Reply:
It's the nomination transfer list + transferable. So the title should be just "Select" or "Use funds from". Thanks for the heads up, actually that makes me think we probably need more information about nomination transfer.

Reply:
> Please correct me if I'm wrong, but why can I select different tokens to stake with even though the chain can only support native tokens?

Are you talking about 'JPYC'? The 'JPYC' is the name of dApp that listed on dApp staking, not ERC20 token. 

<img width="318" alt="image" src="https://user-images.githubusercontent.com/92044428/194987730-4a4d7d5a-dea3-455f-bf53-9c6e9efb8ba9.png">


> The token list also says Select Chain

Thank you, it should be 'Select'. I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/533/commits/0da79fa712331c0fe3da9ce424bb0bf33e15d3b2) commit
![image](https://user-images.githubusercontent.com/92044428/194987153-68c30b88-add9-4243-a2ed-ba7fafa7e52e.png)

Reply:
okay, I realized that it was part of the nomination transfer. I think the label was confusing for me.

Reply:
@Kahonnohak Let me know if you want to change the design or text